binary_transparency/firmware/cmd/ft_personality/internal/http/server_test.go

```diff
@@ -80,6 +81,21 @@ func TestRoot(t *testing.T) {
 }
 
 func TestAddFirmware(t *testing.T) {
+	st, err := base64.StdEncoding.DecodeString("eyJEZXZpY2VJRCI6IlRhbGtpZVRvYXN0ZXIiLCJGaXJtd2FyZVJldmlzaW9uIjoxLCJGaXJtd2FyZUltYWdlU0hBNTEyIjoiMTRxN0JVSnphR1g1UndSU0ZnbkNNTnJBT2k4Mm5RUTZ3aExXa3p1UlFRNEdPWjQzK2NYTWlFTnFNWE56TU1ISTdNc3NMNTgzVFdMM0ZrTXFNdFVQckE9PSIsIkV4cGVjdGVkRmlybXdhcmVNZWFzdXJlbWVudCI6IiIsIkJ1aWxkVGltZXN0YW1wIjoiMjAyMC0xMS0xN1QxMzozMDoxNFoifQ==")
```

Comment:
Since this is "golden" test data, consider adding a

 ```go
func b64(b64 string) []byte {
  r, err := base64.Decode(b64)
  if err panic()
  return r
}
```

then your test is a bit more concise:
```go
func TestXXX(...) {
 st := b64("....base64....")
  ...
}
```

Reply:
ACK